applications/luci-app-simple-adblock/htdocs/luci-static/resources/simple-adblock/status.js

```diff
@@ -0,0 +1,306 @@
+	object: "luci." + pkg.Name,
+	method: "getInitList",
+	params: ["name"],
+});
+
+var getInitStatus = rpc.declare({
+	object: "luci." + pkg.Name,
+	method: "getInitStatus",
+	params: ["name"],
+});
+
+var getPlatformSupport = rpc.declare({
+	object: "luci." + pkg.Name,
+	method: "getPlatformSupport",
+	params: ["name"],
+});
+
+var _setInitAction = rpc.declare({
+	object: "luci." + pkg.Name,
+	method: "setInitAction",
+	params: ["name", "action"],
+	expect: { result: false },
+});
+
+var RPC = {
+	listeners: [],
+	on: function on(event, callback) {
+		var pair = { event: event, callback: callback }
+		this.listeners.push(pair);
+		return function unsubscribe() {
+			this.listeners = this.listeners.filter(function (listener) {
+				return listener !== pair;
+			});
+		}.bind(this);
+	},
+	emit: function emit(event, data) {
+		this.listeners.forEach(function (listener) {
+			if (listener.event === event) {
+				listener.callback(data);
+			}
+		});
+	},
+	getInitList: function getInitList(name) {
+		getInitList(name).then(function (result) {
+			this.emit('getInitList', result);
+		}.bind(this));
+	},
+	getInitStatus: function getInitStatus(name) {
+		getInitStatus(name).then(function (result) {
+			this.emit('getInitStatus', result);
+		}.bind(this));
+	},
+	getPlatformSupport: function getPlatformSupport(name) {
+		getPlatformSupport(name).then(function (result) {
+			this.emit('getPlatformSupport', result);
+		}.bind(this));
+	},
+	setInitAction: function setInitAction(name, action) {
+		_setInitAction(name, action).then(function (result) {
+			this.emit('setInitAction', result);
+		}.bind(this));
+	},
+}
+
+var status = baseclass.extend({
+	render: function () {
+		return Promise.all([
+			L.resolveDefault(getInitStatus(), {}),
+			//			L.resolveDefault(getGateways(), {}),
+		]).then(function (data) {
+			var replyStatus = data[0];
+			//			var replyGateways = data[1];
+			var text ="";
+			var arrPorts = replyStatus[pkg.Name].force_dns_ports;
+			var outputFile = replyStatus[pkg.Name].outputFile;
+			var outputCache = replyStatus[pkg.Name].outputCache;
+			var outputGzip = replyStatus[pkg.Name].outputGzip;
+			var statusTable = [];
+			statusTable["statusNoInstall"] = _("%s is not installed or not found").format(pkg.Name);;
+			statusTable["statusStopped"] = _("Stopped");
+			statusTable["statusStarting"] = _("Starting");
+			statusTable["statusRestarting"] = _("Restarting");
+			statusTable["statusForceReloading"] = _("Force Reloading");
+			statusTable["statusDownloading"] = _("Downloading");
+			statusTable["statusError"] = _("Error");
+			statusTable["statusWarning"] = _("Warning");
+			statusTable["statusFail"] = _("Fail");
+			statusTable["statusSuccess"] = _("Active");
+
+			var header = E('h2', {}, _("Simple AdBlock - Status"))
+			var statusTitle = E('label', { class: 'cbi-value-title' }, _("Service Status"));
+			if (replyStatus[pkg.Name].version) {
+				text += _("Version: %s").format(replyStatus[pkg.Name].version) + " - ";
```

Comment:
```suggestion
				text += _("Version: %s - ").format(replyStatus[pkg.Name].version);
```

Reply:
Thank you. That was intentional, as I want to keep as much "formatting" out of the translatable text.

Reply:
ah, sorry, I didn't notice the `_(` call, I thought it's just used for format call.